sdcm/cluster.py

```diff
@@ -4574,8 +4574,10 @@ def check_cluster_health(self):
             # Don't run health check in case parallel nemesis.
             # TODO: find how to recognize, that nemesis on the node is running
             if self.nemesis_count == 1:
-                for node in self.nodes:
-                    node.check_node_health()
+                with adaptive_timeout(Operations.HEALTHCHECK, node=self.nodes[0],
```

Comment:
Need to use a node without disruption.
For now, if the `self.nodes[0]` is under disruption we will get failure SSHing to it to get scylla.yaml file for is_tablets check.

Reply:
code wraps around healthcheck of all nodes - so we measure overall time.
Node is needed only to get some perf metrics (we don't show them in Argus).
I think there's nothing more to be done here